resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb.erb

```diff
@@ -0,0 +1,376 @@
+#!/bin/ruby
```

Comment:
I think I created this as an erb when I was starting with the scripting, thinking I might end up using some of the ezbake variable interpolation, but those all ended up in the controller, so I can make this not a template.

Ah, yeah. I should've used `#!/usr/bin/env ruby` for more portability.